File: AndroidStudioProjects/ToastDemo/app/src/main/java/com/wambuacooperations/toastdemo/MainActivity.java
package com.wambuacooperations.toastdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    EditText nameEditText;

    public void getName(View view){
        nameEditText=(EditText) findViewById(R.id.nameEditText);
        Toast.makeText(this, "Hello "+nameEditText.getText().toString(), Toast.LENGTH_SHORT).show();
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: JavaFaithful/app/src/main/java/com/wambuacooperations/javafaithful/MainActivity.kt
package com.wambuacooperations.javafaithful

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val john = Person("John", 2000, "Student")
        println("Age is " + john.calcAge())
    }
}